openpaas-james/apps/memory/src/main/java/com/linagora/openpaas/james/app/MemoryServer.java

```diff
@@ -64,9 +64,7 @@ public static void main(String[] args) throws Exception {
             .build();
 
         LOGGER.info("Loading configuration {}", configuration.toString());
-        GuiceJamesServer server = GuiceJamesServer.forConfiguration(configuration)
-            .combineWith(MODULES)
-            .overrideWith(chooseMailbox(configuration.mailboxConfiguration()))
```

Comment:
I was doing 2 time the override, right?

Sorry I did not understood that it was the problem.